examples/text-utilities/src/text-utilities.js

```diff
@@ -95,8 +101,8 @@ export function onAddLineFragments(context) {
   const document = sketch.fromNative(context.document)
 
   // Iterate over each text layer in the selection, calling our addLineFragments function
-  document.selectedLayers.forEach(layer => {
-    if (layer.type === sketch.Types.Text) {
+  document.selectedLayers.layers.forEach(layer => {
+    if (layer.type === String(sketch.Types.Text)) {
```

Comment:
I'm not sure if we should cast to string or just use `==` instead of `===`

Reply:
I think you should just note this in document and let developer to chose the method to resolve the problem, because most eslint config use rule `eqeqeq`

Reply:
yes indeed